Content.Server/GameObjects/EntitySystems/MoverSystem.cs

```diff
@@ -86,7 +86,14 @@ private static void PlayerDetached(PlayerDetachedSystemMessage ev)
             if (ev.Entity.TryGetComponent(out IPhysicsComponent? physics) &&
                 physics.TryGetController(out MoverController controller))
             {
-                controller.StopMoving();
+                var weightless =
+                    !ev.Entity.HasComponent<MovementIgnoreGravityComponent>() &&
+                    PhysicsManager.IsWeightless(ev.Entity.Transform.Coordinates);
+
+                if (!weightless)
+                {
+                    controller.StopMoving();
+                }
```

Comment:
I really hope this will be improved once we have force controllers..

Reply:
This could be completely removed with force controllers.